### PR TITLE
[docs] Add description tag to pages where it is missing

### PR DESCRIPTION
**Pull request by amandeepmittal**
# Why

Follow up to #24669 (Initially, I left out these pages as I wasn't sure if we should keep them or remove them, but let's add the description tag for now)

# How

Add missing description tags.

# Test Plan

Run docs locally, go to pages individually and look for the `description` meta tag.

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

File: docs/pages/troubleshooting/clear-cache-macos-linux.mdx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Clearing bundler caches on macOS and Linux
+description: Learn about the measures you can take to clear bundler caches on macOS and Linux when using Expo.
```
Comment (Simek) [inline docs/pages/troubleshooting/clear-cache-macos-linux.mdx:3]: I don't like that much the `measures you can take` part, it seems to make description less concise and the term itself do not add any information. But in the other hand, description without this addition is quite similar to the page title.

Another part is the "with Expo" suffix, looking at the page content, we also guide people how to clear the project when they are not using Expo, but RN CLI.

Below are two suggestion, but feel free to alter the description in other way, those are just examples.

```suggestion
description: Learn how to clear bundler caches on macOS and Linux when building React Native app.
```

```suggestion
description: Learn how to clear React Native app bundler caches on macOS and Linux.
```

Comment (amandeepmittal) [reply docs/pages/troubleshooting/clear-cache-macos-linux.mdx:3]: Great points!

I didn't give it much of thought initially. Have removed the phrasing you pointed out from all the pages.

Comment (amandeepmittal) [reply docs/pages/troubleshooting/clear-cache-macos-linux.mdx]: @Simek, I've updated the description tag for the pages under the `/troubleshooting/`. I've also fixed formatting issues (such as not using `Terminal` component) and some minor grammatical issues.

**Review by Simek — APPROVED** [review]
Thank you for the updates, LGTM! 👍

**Pull request merged**